utils/make_package.py

```diff
@@ -0,0 +1,82 @@
+﻿#!/usr/bin/env python
+
+import sys
+import argparse
+import os
+import subprocess
+import platform
+import io
+import string
+import urllib2
+import shutil
+import stat
+
+def run(args):
+  nugetFolder = os.path.join(args.target, ".nuget")
+  print("\nEnsuring folder: %s" % nugetFolder )
+  if not os.path.exists(nugetFolder):
+    os.makedirs(nugetFolder)
+  
+  nugetExe = os.path.join(nugetFolder, "nuget.exe")
+  if not os.path.exists(nugetExe):
+    nugetOrg = "http://nuget.org/nuget.exe"
```

Comment:
Yep. I think so. I looked at build.sh in coreclr which also downloads this thing.
I tested this script in Linux, which crated a packages (using the nuspec file for Linux)